sdks/wasm/packager.cs

```diff
@@ -300,7 +300,7 @@ enum CopyType
 			Import (resolved, kind);
 		}
 		if (add_binding)
-			Import (ResolveFramework (BINDINGS_ASM_NAME + ".dll"), AssemblyKind.Framework);
+			Import (ResolveBcl (BINDINGS_ASM_NAME + ".dll"), AssemblyKind.Framework);
 
```

Comment:
Is the ResolveFramework code needed any more ?

Reply:
Will remove it.

Reply:
addressed in packager.cs